frontend/src/components/profile/SeeMatchHistoryBtn.tsx

```diff
@@ -0,0 +1,12 @@
+import { Link } from 'react-router-dom'
+import styles from './SeeMatchHistoryBtn.module.css'
+
+const SeeMatchHistoryBtn = () => {
+    return (
+        <Link to="/history" className={styles.btn}>
+            see match history
+        </Link>
+    )
+}
+
+export default SeeMatchHistory
```

Comment:
So add Btn here too ?
```suggestion
export default SeeMatchHistoryBtn
```

Reply:
Of course!